src/UI/SignTransactionsModals/components/ConfirmationScreen.tsx

```diff
@@ -52,8 +52,7 @@ export const ConfirmationScreen = ({
 
   const transactionStatusToastProps: TransactionStatusToastType = {
     signError,
-    canceledTransactionsMessage,
-    onDelete: handleClose
```

Comment:
onDelete: handleClose

not needed anymore ?

Reply:
yes, needed for the fix. onDelete is optional and is kept and passed by other components, but not in this case